incubator/hnc/pkg/reconcilers/hierarchy_config.go

```diff
@@ -365,17 +382,21 @@ func (r *HierarchyConfigReconciler) syncChildren(log logr.Logger, inst *api.Hier
 				msg = fmt.Sprintf("required subnamespace %s does not exist", cn)
 			}
 		} else {
-			// Someone else got it first. This should never happen if the validator is working correctly.
-			other := cns.RequiredChildOf
-			if other == "" {
-				other = cns.Parent().Name()
+			if !r.HNSReconcilerEnabled {
```

Comment:
Worth a comment here to say what happens if we're using HNS, even if it's just a "TODO."